lzy/allocator/src/main/java/ai/lzy/allocator/services/AllocatorService.java

```diff
@@ -1000,4 +1002,17 @@ private Volume.AccessMode validateAccessMode(VolumeApi.DiskVolumeType.AccessMode
             default -> Volume.AccessMode.READ_WRITE_ONCE;
         };
     }
+
+    @Nullable
+    private DiskVolumeDescription.StorageClass validateStorageClass(
+        VolumeApi.DiskVolumeType.StorageClass storageClass
+    ) {
+        return switch (storageClass) {
+            case DEFAULT -> null; //todo think about it
```

Comment:
That won't work because we use common proto messages to create VM volumes and dynamic mount. If we prohibit default values, we will break compatibility

Reply:
Good point, you're right :+1: 
But... until we don't have rolling restart - we can ignore the backward compatibility :)